### PR TITLE
[New Rule] Suspicious Inter-Process Communication via Outlook

### PR DESCRIPTION
**Pull request by Samirbous**
Detect Inter-Process communication via COM with Outlook, this could be a sign of Local Email Collection (T1114.001) from outlook or other form of code execution: 

image

### CODE REVIEW

File: rules/windows/collection_email_outlook_mailbox_via_com.toml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+Detects Inter-Process Communication with Outlook via Component Object Model from an unusual process. Adversaries may target user email to collect
+sensitive information or send email on their behalf via API.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Suspicious Inter-Process Communication via Outlook"
+references = [
+    "https://github.com/dafthack/MailSniper/blob/master/MailSniper.ps1",
```
Comment (w0rk3r) [inline rules/windows/collection_email_outlook_mailbox_via_com.toml:21]:
```suggestion
```

File: rules/windows/collection_email_outlook_mailbox_via_com.toml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+[[rule.threat.technique.subtechnique]]
+id = "T1114.001"
+name = "Local Email Collection"
+reference = "https://attack.mitre.org/techniques/T1114/001/"
+
+[[rule.threat.technique.subtechnique]]
+id = "T1114.002"
+name = "Remote Email Collection"
+reference = "https://attack.mitre.org/techniques/T1114/002/"
```
Comment (w0rk3r) [inline rules/windows/collection_email_outlook_mailbox_via_com.toml:53]:
```suggestion
```

**Review by w0rk3r — APPROVED** [review]
🧙🏼

File: rules/windows/collection_email_outlook_mailbox_via_com.toml [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+ not process.Ext.effective_parent.executable : ("?:\\Program Files\\*", "?:\\Program Files (x86)\\*")
+'''
+
+
+[[rule.threat]]
+framework = "MITRE ATT&CK"
+[[rule.threat.technique]]
+id = "T1114"
+name = "Email Collection"
+reference = "https://attack.mitre.org/techniques/T1114/002/"
```
Comment (w0rk3r) [inline rules/windows/collection_email_outlook_mailbox_via_com.toml:43]:
```suggestion
reference = "https://attack.mitre.org/techniques/T1114/"
```

**Review by Aegrah — APPROVED** [review]

**Review by DefSecSentinel — APPROVED** [review]
LGTM

**Review by terrancedejesus — APPROVED** [review]

**Pull request merged**